### ISSUE TITLE
Helm chart is broken

### ISSUE THREAD

**Issue by pzduniak** [mailgun/gubernator#130]
https://github.com/mailgun/gubernator/blob/master/deploy/helm/templates/_helpers.tpl#L101-L102

Env vars must be strings, this inserts a number. The value needs to be wrapped in `"`s.

I'm not fully convinced this has ever worked :) I'd fix this, but legally it would get complicated.

**Comment by pzduniak** [comment]
Another one!

```yaml
  metrics:
    - type: Resource
      resource:
        name: cpu
        target:
          type: Utilization
          averageUtilization: {{ .Values.gubernator.autoscaling.cpuAverageUsage }}
```

meanwhile in `values.yaml`:
```yaml
gubernator:

  fullnameOverride: {}
  nameOverride: {}
  priorityClassName: {}

  autoscaling:
    enabled: true
    minReplicas: 2
    maxReplicas: 10
    cpuAverageUtilization: 50
```
note `cpuAverageUsage` vs `cpuAverageUtilization`

**Comment by thrawn01** [comment]
I do not use helm, the current helm chart is provided as a convenience to others who would make use of it. If you have some specific changes would you mind making a PR and I'll merge it.

**Comment by Baliedge** [comment]
@pzduniak What version(s) of Kubernetes and Helm have you tested on? What versions had problems before these changes were made?

**Comment by pzduniak** [comment]
We're running 1.21, I think at first I started out on 1.19.

I don't think the Helm chart was even usable, it looked like a ton of accidental typos broke it. Let's assume that the Helm chart was designed for 1.19, which was released on 2020-08-26:
1. `_helpers.tpl` declared an env var value as a number - [I don't think think that it was ever possible to pass numbers as the `value`](https://v1-19.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#envvar-v1-core)
2. `cpuAverageUsage` simply does not exist in `values.yaml`. I think whoever was testing it passed it as a CLI option, assuming that (1) was somehow worked around.
3. `nodeSelector` could've been the configuration used in the test cluster. That's alright, but I don't want to add node labels just because some project requires them.
